RecoTracker/IterativeTracking/python/JetCoreRegionalStep_cff.py

```diff
@@ -95,14 +137,26 @@
     seedPairPenalty     = 0,
     minPt               = 0.1
 )
-
-from Configuration.ProcessModifiers.seedingDeepCore_cff import seedingDeepCore
-seedingDeepCore.toModify(jetCoreRegionalStepTrajectoryFilter,
+jetCoreRegionalStepBarrelTrajectoryFilter = TrackingTools.TrajectoryFiltering.TrajectoryFilter_cff.CkfBaseTrajectoryFilter_block.clone(
     minimumNumberOfHits = 2,
     maxConsecLostHits   = 2,
     maxLostHitsFraction = 1.1,
-    minPt               = 0.9
-    )
+    seedPairPenalty     = 0,
+    minPt               = 0.9 ## should it be slightly decrease ?
```

Comment:
> please remind me why these settings are justified compared to the endcap part
> 
>     * `minimumNumberOfHits = 4` vs 2 and `maxConsecLostHits   = 2`: are the hits counted differently?
> 
>     * `minPt = 0.9` vs 0.1: this looks like a physics optimization; if it's acceptable to have a cut at 0.9 in the barrel, why not keep the same in the endcap?
> 
> 
> For simpler maintenance it would be much better to just make the seeds, merge them, and then pass to the same building and fitting modules instead of splitting.

please comment